thunder.install

```diff
@@ -92,6 +92,36 @@ function _thunder_mark_update_checklist($update_id, $successful, UpdateLogger $u
   }
 }
 
+/**
+ * Installs a module without calling any hooks nor config imports.
+ *
+ * @param string $module
+ *   The module to install.
+ */
+function _thunder_bare_install($module) {
+  // Load a new config object for each iteration, otherwise changes made
+  // in hook_install() are not reflected in $extension_config.
+  $extension_config = \Drupal::configFactory()->getEditable('core.extension');
+
+  $extension_config
+    ->set("module.$module", 0)
+    ->set('module', module_config_sort($extension_config->get('module')))
+    ->save(TRUE);
+
+  // Set the schema version to the number of the last update provided by
+  // the module, or the minimum core schema version.
+  $version = \Drupal::CORE_MINIMUM_SCHEMA_VERSION;
+  $versions = drupal_get_schema_versions($module);
+  if ($versions) {
+    $version = max(max($versions), $version);
+  }
+
+  drupal_set_installed_schema_version($module, $version);
+
+  $kernel = \Drupal::service('kernel');
```

Comment:
This bit doesn't look that right - I think we need to copy the module installer and do the update modules. That said this might work - we need to work out what happens to the autoloader.